org-formation/710-tgw/_tasks.yaml

```diff
@@ -149,7 +149,7 @@ Mappings:
       '055273631518': 'snowflakevpc'   # org-sagebase-scicomp
       '420786776710': 'bridge-aux'   # org-sagebase-bridgedev
       '325565585839': 'synapse-ops-vpc-v2' # org-sagebase-synapseprod
-      '449435941126': 'sagevpc' # org-sagebase-synapsedev
+      '449435941126': 'vpc' # org-sagebase-synapsedev
```

Comment:
could it have been renamed manually from the aws console?  can you figure out what the name should be from the template that was used to create the VPC?

Reply:
It was passed as param.